test/unit/webrtc.test.ts

```diff
@@ -1849,9 +1850,74 @@ describe('handleGenesysOffer', () => {
       expect(session.setRemoteDescription).toHaveBeenCalled();
       expect(webrtc['webrtcSessions'].length).toEqual(1);
     });
+    const spy = webrtc['handleGenesysRenegotiate'] = jest.fn();
 
     await webrtc['handleGenesysOffer'](iq);
-    expect.assertions(2);
+    expect(spy).not.toHaveBeenCalled();
+    expect.assertions(3);
+  });
+
+  it('should renegotiate if there is an existing session', async () => {
+    const iq: IQ = {
+      type: 'set',
+      from: '+155555555@org.gjoll.test',
+      genesysWebrtc: {
+        jsonrpc: '2.0',
+        method: 'offer',
+        params: {
+          sessionId: 'session24',
+          conversationId: 'cid',
+          sdp: 'a=ice-ufrag:asdfasdf\r\na=ice-pwd:fdsafdsa\r\n'
+        }
+      }
+    };
+
+    webrtc.on('incomingRtcSession', (session: GenesysCloudMediaSession) => {
+      expect(session.setRemoteDescription).toHaveBeenCalled();
+      expect(webrtc['webrtcSessions'].length).toEqual(1);
+    });
```

Comment:
Just dropping a note about what we briefly chatted about last week; I think we need `expect.assertions` to ensure these are being called.